src/main/scala/outwatch/dom/Compat.scala

```diff
@@ -29,8 +29,30 @@ trait Handlers {
 
 object Handlers extends Handlers
 
-trait ReflectedAttrsCompat[RA[_, _]] { self: reflectedAttrs.ReflectedAttrs[RA] =>
-  lazy val `class`: RA[String, String] = className
+trait AttributesCompat {
+  lazy val `class` = className
 
-  lazy val `for`: RA[String, String] = forId
+  @deprecated("Use forId instead", "0.11.0")
+  lazy val `for` = forId
```

Comment:
Why should we deprecate this? Maybe scala-dom-types could include it and also `` `class` ``? It already includes `` `type` ``.

@raquo

Reply:
Would be fine with me, I already opened a [PR](https://github.com/raquo/scala-dom-types/pull/14) for `class`, but maybe we can even have both? :)